spec/swagger.yaml

```diff
@@ -136,6 +136,28 @@ tags:
 
       For each string in a Smartling project, there is a unique identifier created by hashing together the parsed string text, variant metadata, and in some cases, namespace. When adding strings via the Strings API, each string will be unique. If you try to create a string with identical variant metadata and string text to a string that already exists in the project, the string will be overwritten.
 
+  - name: "Jobs Batches"
```

Comment:
I would like to propose to move it right after `Jobs`

Reply:
good for me